test/functional/helpers/session.js

```diff
@@ -137,7 +138,22 @@ async function initSession (caps) {
     }, caps);
   }
 
-  let serverRes = await driver.init(caps);
+  let error;
+  const serverRes = await retryInterval(2, 1000, async function () {
+    try {
+      return await driver.init(caps);
+    } catch (err) {
+      if (err.message.includes('failed to finish booting')) {
+        // sometimes the sim doesn't start, usually because of "Data Migration"
```

Comment:
can this be fixed on the lower level (node-simctl)?

Reply:
I'm hoping that it is just because of older betas of Xcode 11. It can really be taken out, and things adjusted when Travis gets the full version when it is released.